sdk/core/core-http/src/credentials/accessTokenRefresher.ts

```diff
@@ -0,0 +1,67 @@
+import { AccessToken, TokenCredential, GetTokenOptions } from "@azure/core-auth";
+
+/**
+ * Helps the core-http token authentication policies with requesting a new token if we're not currently waiting for a new token.
+ */
+export class AccessTokenRefresher {
+  private promise: Promise<AccessToken | undefined> | undefined;
+  private lastCalled: number | undefined;
```

Comment:
instead of `undefined`, a minimum number like 0 or beginning of the time can be used here instead?

Reply:
I'll assign 0